src/test/java/org/assertj/core/api/double_/DoubleAssert_isNegativeInfinity_Test.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api.double_;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.DoubleAssert;
+import org.assertj.core.api.DoubleAssertBaseTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+/**
+ * Tests for <code>{@link DoubleAssert#isNegativeInfinity()}</code>.
+ * 
+ * @author Jin Kwon
+ */
+class DoubleAssert_isNegativeInfinity_Test extends DoubleAssertBaseTest {
+
+  @Override
+  protected DoubleAssert invoke_api_method() {
+    return assertions.isNegativeInfinity();
+  }
+
+  @Override
+  protected void verify_internal_effects() {
+    verify(doubles).assertIsNegativeInfinity(getInfo(assertions), getActual(assertions));
+  }
+
+  @Test
+  void should_pass_if_actual_is_NEGATIVE_INFINITY() {
+    final double actual = Float.NEGATIVE_INFINITY;
+    assertThat(actual).isNegativeInfinity();
+  }
+
+  @MethodSource({ "org.assertj.core.api.double_.DoubleAssertTestParameters#zeros" })
+  @ParameterizedTest
+  void should_fail_if_actual_is_zero(final double actual) {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat(actual).isNegativeInfinity())
+                                                   .withMessageContainingAll("" + actual, "" + Double.NEGATIVE_INFINITY);
```

Comment:
Use `expectAssertionError` as stated in the contributing guide, `OptionalAssert_containsInstanceOf_Test` is a good example.

Reply:
Same comment applies to other tests too in this class and the other test classes